chart/Chart.yaml

```diff
@@ -3,6 +3,7 @@ name: rancher
 description: Install Rancher Server to manage Kubernetes clusters across providers.
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: < 1.20.0
```

Comment:
How are we going to keep track of this over time ? Question for wider team.

Reply:
As part of the introducing new minor versions into Rancher, we should add this as a step.

Reply:
@StrongMonkey As part of adding k8s 1.20 into Rancher v2.5 line, please update this kubeVersion. Can you also add it into the process docs about introducing new k8s versions. 

@paynejacob Can you please make a PR into the release/v2.5 branch and master branch?